src/TodoInput.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function TodoInput({ todoInput, onChangeContent, onClickAdd }) {
+  return (
+    <>
+      <input
+        type="text"
+        placeholder="할 일을 입력해 주세요"
+        value={todoInput}
+        onChange={(event) => onChangeContent(event.target.value)}
```

Comment:
```suggestion
        onChange={onChangeContent}
```

선언부에서 event를 받는 방식이면 갈끔하겠네요.